lib/web_ui/lib/src/engine/canvaskit/painting.dart

```diff
@@ -22,115 +22,82 @@ import 'shader.dart';
 /// This class is backed by a Skia object that must be explicitly
 /// deleted to avoid a memory leak. This is done by extending [SkiaObject].
 class CkPaint implements ui.Paint {
```

Comment:
Since we will have to do something similar for skwasm, do you think it might be worthwhile to pull this out of the canvaskit-specific code and instead just make it a renderer-agnostic class that I could reuse in skwasm? And perhaps just making an extension method for `toSkPaint` or whatever?

Reply:
Not a bad idea! I'm not going to do it in this PR, since `CkPaint` has direct dependencies on `canvaskit_api.dart` and many other `Ck*` classes. But it can be done as a follow-up when `SkwasmPaint` moves to this model. I filed https://github.com/flutter/flutter/issues/154281 and will leave a TODO.